charts/ocis/templates/storagesystem/deployment.yaml

```diff
@@ -78,27 +78,27 @@ spec:
             - name: STORAGE_SYSTEM_HTTP_ADDR
               value: 0.0.0.0:9216
             - name: STORAGE_SYSTEM_DATA_SERVER_URL
-              value: http://storage-system:9216/data
+              value: http://{{ .appName }}:9216/data
```

Comment:
I just did a plain replacement here right now, but wouldn't it make sense to use `localhost` here as this pod is the storage system?

Reply:
Ok.